testscripts/ld/numpy/devbox.json

```diff
@@ -0,0 +1,20 @@
+{
+  "packages": {
+    "python": "latest",
+  },
+  "shell": {
+    "init_hook": [
+      // Make sure we're using a new virtualenv in case the OS or Python version
+      // changed.
+      "export VENV_DIR=$(mktemp -d)",
+      "python -m venv $VENV_DIR",
+    ],
+    "scripts": {
+      "test": [
```

Comment:
for these to run automatically theyneed to be called `run_test`. Maybe want to call it `run_test_disabled` ? Or add an env var so it exits 0 by default if env var is not set